services/api/service.go

```diff
@@ -1595,6 +1604,185 @@ func (api *RelayAPI) checkBuilderEntry(w http.ResponseWriter, log *logrus.Entry,
 	return builderEntry, true
 }
 
+// Checks the quality of the TOB txs, if it is the txs expected in a TOB
+func (api *RelayAPI) checkTobTxsStateInterference(txs []*types.Transaction) error {
+	if len(txs) > 2 {
+		return fmt.Errorf("we support only 1 tx on the TOB currently, got %d", len(txs))
+	}
```

Comment:
maybe add a comment saying the second is a payment ?

Reply:
yes!